package.json

```diff
@@ -68,16 +100,31 @@
     "grunt-plato": "0.1.4",
     "grunt-preprocess": "4.0.0",
     "grunt-template": "0.2.3",
```

Comment:
@thejameskyle I can see how all would be ideal, but these are **big** PRs. Removing grunt will be the gruntfile, docs, test helpers, src helpers... 

And, if we're using gulp now to do the main things all of the new code is being executed.

Further, I'm sure there's plenty here that could be reviewed :)